src/docs/development/tools/android-studio.md

```diff
@@ -129,31 +132,56 @@ of your changes almost instantly with the _hot reload_ feature. See
-  [Flutter Inspector](/docs/development/tools/inspector).
-* **Toggle Debug Paint**: Add visual debugging hints to the rendering displaying
-   borders, padding, alignment, and spacers.
-* **Toggle Platform Mode**: Toggle between rendering for Android or iOS.
-* **Toggle Performance Overlay**: Show performance graphs for the GPU & CPU threads.
-* **Open Timeline View**: Analyze activity of the application as it runs.
-* **Open Observatory**: A profiler for Dart applications.
-
-Also available in the additional actions menu:
-
-* **Show Paint Baselines**: Causes each RenderBox to paint a line at each of its
-  baselines.
-* **Enable Repaint Rainbow**: Shows rotating colors on layers when repainting.
-* **Enable Slow Animations**: Slows down animations to enable visual inspection.
-* **Hide Debug Banner**: Hides the debug banner even when running a debug build.
-
+* **Enable select widget mode**: Select a widget on the device to inspect
+  in the [Flutter Inspector](/docs/development/tools/inspector).
+* **Refresh widget info**: Redisplay the current widget info.
+* **Show/hide performance overlay**: Toggle display of performance graphs
+  for the GPU & CPU threads.
+* **Toggle platform mode**: Toggle between rendering for Android or iOS.
+* **Show debug paint**: Add visual debugging hints to the rendering
+  displaying borders, padding, alignment, and spacers.
+* **Show paint baselines**: Cause each RenderBox to paint a line at each
+  of its baselines.
+* **Enable slow animations**: Slow down animations to enable visual
+  inspection.
+
+Also available in the "more actions" menu:
+
+* **Show Repaint Rainbow**: Shows rotating colors on layers when repainting.
+* **Hide Debug Mode Banner**: Hides the debug banner even when running a
+  debug build.
+* **Auto horizontal scroll**: XXX - TBD
+* **Highlight nodes displayed in both trees**: XXX - TBD
+* **Open Timeline View**: Analyzes activity of the application as it runs.
+* **Open Observatory**: Profiles Dart applications.
+
+### Show performance data
+
+To view the performance data, including the widget rebuild information,
+start the app with **Debug**, and then open the Performance tool window
+using **View > Tool Windows > Flutter Performance**.
+
+![Flutter performance window]({% asset tools/android-studio/widget-rebuild-info.png @path %})
+
+To see the stats about which widgets are being rebuilt, and how often,
+click **Show widget rebuild information** in the **Performance** pane.
+You can see how many times a widget was rebuilt in the last frame.
+For a high number of rebuilds, the exact count shows in the second column
+from the right, and a yellow spinning circle displays. 
+The column to the far right shows how many times a widget was rebuilt
+since entering the current screen. The app shown in this screenshot
+demonstrates poor performance.
+For widgets that weren't rebuilt, a solid grey circle displays.
+Otherwise, a grey spinning circle displays.
```

Comment:
Also call out the following:
The purpose of this tool is to help you be aware of when widgets are rebuilding that you might not expect are rebuilding by just looking at the code. If widgets are rebuilding that you didn't expect it is probably a sign you should refactor your code by splitting up large build methods into multiple widgets.
Also provide the 4 performance tips we give if we detect obvious performance problems here. 
https://github.com/InMatrix/perf_diagnosis_demo
has 4 good concrete examples that trigger performance tips and show the sorts of cases this feature is trying to solve.

Also call out that you can click on the lines in the table to navigate to the source line where the widget was created. Similarly you might want to call out that the spinning icons also show up in the gutter of the text file to help you visualize what rebuilds are happening when code is running.

Additional things to call out: the fact that there are a number of rebuilds in itself does not indicate a problem. Typically should should only worry about excessive rebuilds if you have already run the app in profile mode and verified that the performance is not what you want.
Also call out that the widget rebuild information is only available in debug builds.